app/src/main/java/com/gdsctsec/smartt/database/Database.kt

```diff
@@ -0,0 +1,35 @@
+package com.gdsctsec.smartt.database
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+@Database(entities = [TimeTable::class], version = 1, exportSchema = false)
+abstract class LectureDatabase : RoomDatabase(){
```

Comment:
can you rename this as `AppDatabase`?, that sounds intuitive

Reply:
Ok cool